product_contract/migrations/18.0.1.0.0/pre-migrate.py

```diff
@@ -0,0 +1,17 @@
+# Copyright 2025 ACSONE SA/NV
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+from openupgradelib import openupgrade
+
+
+def migrate(cr, version):
+    openupgrade.rename_fields(
+        cr,
```

Comment:
See https://github.com/OCA/openupgradelib/blob/237ee5cd4c5304a578e9e04844fa9c13de55c357/openupgradelib/openupgrade.py#L675C1-L675C51
```suggestion
        env,
```